k8s/base.py

```diff
@@ -273,6 +252,40 @@ def _api_name(name):
     return name[1:] if name.startswith("_") else name
 
 
+def _label_selector(labels):
+    """
+    Build a labelSelector string from a collection of key/values. The parameter can be either
+    a dict, or a list of (key, value) tuples (this allows for repeating a key).
+
+    The keys/values are used to build the `labelSelector` parameter to the API,
+    and supports all the operations of the API through the use of :py:class:`~k8s.base.LabelSelector`.
+
+    Each key is a label name. The value defines which operation to perform.
+    Operations that take a single string value:
+
+        - :py:class:`~k8s.base.Equality`
+        - :py:class:`~k8s.base.Inequality`
+
+    A plain string will automatically be wrapped by :py:class:`~k8s.base.Equality` for compatability
+    with older versions of this method.
+
+    Operations that take a sequence of string values:
+
+        - :py:class:`~k8s.base.In`
+        - :py:class:`~k8s.base.NotIn`
+
+    Operations that takes no value:
+
+        - :py:class:`~k8s.base.Exists`
+    """
+
+    if hasattr(labels, "items"):
+        labels = sorted(labels.items(), key=lambda kv: kv[0])
+
+    return ",".join("{}{}".format(k, v if isinstance(v, LabelSelector) else Equality(v))
+                            for k, v in labels)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [continuation line over-indented for visual indent (E127)](https://app.codacy.com/manual/fiaas/k8s/pullRequest?prid=5606786)